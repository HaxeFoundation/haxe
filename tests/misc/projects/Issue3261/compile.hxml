--display Main.hx@147