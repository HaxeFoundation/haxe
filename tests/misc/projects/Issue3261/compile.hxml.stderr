<type>
b : Int -&gt; String
</type>

